src/test/txrebroadcast_tests.cpp

```diff
@@ -114,7 +160,7 @@ BOOST_AUTO_TEST_CASE(max_rebroadcast)
     SetMockTime(current_time);
     // Then check that it gets returned for rebroadacst
     candidates = tx_rebroadcast.GetRebroadcastTransactions();
-    BOOST_CHECK_EQUAL(candidates.size(), 1);
+    BOOST_REQUIRE_EQUAL(candidates.size(), 1);
```

Comment:
In 0f13d876977d6a83549fa40f68a73308ba10d95d "[test] Add unit test for the fee rate cache"

Seems like this change should be part of the previous commit.